anclafx/provide.go

```diff
@@ -19,7 +19,6 @@ func Provide() fx.Option {
 			ancla.ProvideListener,
 			ancla.ProvideDefaultListenerWatchers,
 			chrysom.ProvideBasicClient,
-			chrysom.ProvideDefaultListenerReader,
```

Comment:
ancla will  solely use the argus client until we discover a use case for swappable db clients